content/en/docs/Getting Started/pipelines.md

```diff
@@ -32,9 +32,10 @@ spec:
   steps:
     - name: goodbye
       image: ubuntu
-      script: |
-        set -e
-        echo "Goodbye World!"
+      command:
```

Comment:
The version we want to recommend to users is actually the `script` one, so I would update other docs to match that

Reply:
Got it, reverted back to a script and updated it so that we get the correct output.